src/transforms/push.ts

```diff
@@ -112,7 +120,6 @@ export const processPush = async (github: GitHubAPI, payload, rootLogger: Logger
 		const webhookReceived = payload.webhookReceived || undefined;
 
 		const log = rootLogger.child({
-			name: PUSH_LOGGER_NAME, // overriding even though it is provided worker; webserver processed the webhooks synchronously, too
```

Comment:
I removed the name override from here because:
1. It was failing when we pass logger from the sqs handler code 
2. We want to differentiate from where this code was called by logger name (From the webhook request handler, from redis message handling or sqs message handling)